test/room_test.rb

```diff
@@ -21,4 +21,17 @@ def test_it_can_get_area
     assert_equal 130, room1.area
     assert_equal 180, room2.area
   end
+
+  def test_is_painted?
+    skip
```

Comment:
Not sure why you have this test and the next one skipped? Both methods are working.